staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -146,6 +146,7 @@ func NewDrainCmdOptions(f cmdutil.Factory, ioStreams genericclioptions.IOStreams
 		IOStreams:  ioStreams,
 		drainer: &drain.Helper{
 			GracePeriodSeconds: -1,
+			IgnoreErrors:       false,
```

Comment:
This is the default

Reply:
Yes, default not ignore errors, this is same with now command behavior.